services/core-web/src/components/noticeOfWork/applications/administrative/NOWApplicationAdministrative.js

```diff
@@ -67,12 +65,12 @@ export const NOWApplicationAdministrative = (props) => {
         <ScrollContentWrapper id="generated-documents" title="Application Export Files">
           <NOWDocuments
             documents={props.noticeOfWork.documents.filter(
-              ({ now_application_document_type_code }) =>
-                exportedDocuments.includes(now_application_document_type_code)
+              ({ now_application_document_sub_type_code }) =>
+                now_application_document_sub_type_code === "AEF"
             )}
             isViewMode
             disclaimerText="This table shows all of the PDF files created from the edited Notice of Work form."
```

Comment:
Since this will also now show PMT and PMA documents, do we want to adjust this text?

Reply:
That's a good suggestion, Does this table include all the generated documents? Client acknowledgment, permit enclosed, withdrawal, etc?

Reply:
Nope, those letters are classified as GDO and will appear in the Government Documents section. They only classified NTR, PMT, and PMA as "Application Export Files".

Reply:
Makes sense